src/components/forms/SubmitButton.tsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import { wallet } from '../../services/near';
+import { FaInfoCircle } from 'react-icons/fa';
+import ReactTooltip from 'react-tooltip';
 
 interface SubmitButtonProps {
   text: string;
   disabled?: boolean;
   onClick?: (event: React.MouseEvent) => void;
+  info: string;
```

Comment:
This should be optional. And the info icon should only appear if you pass this prop